llvmlite/binding/module.py

```diff
@@ -240,6 +240,24 @@ def _next(self):
         return ffi.lib.LLVMPY_FunctionsIterNext(self)
 
 
+class _BlocksIterator(_Iterator):
+
+    def _dispose(self):
+        self._capi.LLVMPY_DisposeBlocksIter(self)
+
+    def _next(self):
+        return ffi.lib.LLVMPY_BlocksIterNext(self)
+
+
+class _InstructionsIterator(_Iterator):
+
+    def _dispose(self):
+        self._capi.LLVMPY_DisposeInstructionsIter(self)
+
+    def _next(self):
+        return ffi.lib.LLVMPY_InstructionsIterNext(self)
```

Comment:
Same as above.  Need to define `.restype`.

Reply:
Fixed.